src/Evolution/Systems/GrMhd/ValenciaDivClean/Tags.hpp

```diff
@@ -32,7 +32,7 @@ struct TildeD : db::SimpleTag {
   using type = Scalar<DataVector>;
 };
 
-/// The densitized electron_fraction \f${\tilde Ye}\f$
+/// The densitized electron fraction times rest mass density \f${\tilde Y}_e\f$
```

Comment:
How about saying `// The densitized electron number density \f${\tilde Y}_e\f$`
since rho*Y_e is the electron number density?

Reply:
maybe `.. electron number density times the baryon mass` ?

Reply:
that works, too!